guides/source/routing.md

```diff
@@ -863,29 +918,37 @@ get 'books/*section/:title', to: 'books#show'
 
 would match `books/some/section/last-words-a-memoir` with `params[:section]` equals `'some/section'`, and `params[:title]` equals `'last-words-a-memoir'`.
 
-Technically, a route can have even more than one wildcard segment. The matcher assigns segments to parameters in an intuitive way. For example:
+Technically, a route can have even more than one wildcard segment. The matcher assigns segments to parameters in the order they occur. For example:
 
 ```ruby
 get '*a/foo/*b', to: 'test#index'
 ```
 
 would match `zoo/woo/foo/bar/baz` with `params[:a]` equals `'zoo/woo'`, and `params[:b]` equals `'bar/baz'`.
 
-NOTE: By requesting `'/foo/bar.json'`, your `params[:pages]` will be equal to `'foo/bar'` with the request format of JSON. If you want the old 3.0.x behavior back, you could supply `format: false` like this:
+### Format Segments
+
+Given this route definition:
+
+```ruby
+get '*pages', to: 'pages#show'
+```
+
+By requesting `'/foo/bar.json'`, your `params[:pages]` will be equal to `'foo/bar'` with the request format of JSON. If you want to match URLs without an explicit format and ignore URLs that include a format extension, you could supply `format: false` like this:
 
 ```ruby
 get '*pages', to: 'pages#show', format: false
 ```
 
-NOTE: If you want to make the format segment mandatory, so it cannot be omitted, you can supply `format: true` like this:
+If you want to make the format segment mandatory, so it cannot be omitted, you can supply `format: true` like this:
```

Comment:
Is this the default behavior? Is there a default behavior?

Reply:
Added "The default behavior with `format` is that if included Rails automatically captures it from the URL and includes it in params[:format], but `format` is not required in a URL."